### PR TITLE
Create LCD Block Subsystem.md

### PR DESCRIPTION
**Pull request by abdoulm366**

### CODE REVIEW

**Review by JesseTNRoberts — COMMENTED** [review]
I don't understand what the purpose of this is. What is this supposed to do and where is this LCD located?

**Review by JesseTNRoberts — COMMENTED** [review]
Feedback given in person

**Review by JesseTNRoberts — COMMENTED** [review]
It seems that I have not been requested to review this system. But I was set as an assignee. That is not the correct action. Be sure to request review when you want feedback.

**Review by JesseTNRoberts — COMMENTED** [review]
solderless breadboards aren't allowed in capstone designs.

This document is messy. I don't see constraints. The images are good to include. However, they need to be scaled down and placed in a grid. There's no reason to have them take up so much space.

**Review by JesseTNRoberts — COMMENTED** [review]
Comments not addressed. Do not resubmit until the identified issues have been fixed.

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+## Subsystem Function:
+
+The purpose of this subsystem is to help ease the user's understanding of the kit's functionalities, provide a digital user manual, provide the users with instructions or guided learning via challenges and lessons and enable a digital interaction with the car. Hence, this LCD Block Subsystem serves as the one of the primary interaction point between the user and the device as a whole. 
+
+## Constraints/Specifications:
+
+| Description       | Constraints/Specification:                                                                           | Source                         |
+|-------------------|------------------------------------------------------------------------------------------------------|--------------------------------|
+| Visibility        | The LCD must provide clear visibility in varying light conditions.                                   |  Interaction Design principles [3]             |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/LCD Block Subsystem.md]: How varying? How do we know that the chosen display can accomplish this?

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+| User Interface    | The menu should be intuitive, allowing easy navigation for all ages.                                 | User interface design [4]       |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/LCD Block Subsystem.md]: How will we measure intuitiveness?

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+| Interactivity     | The system must register input from the keypad within 300ms to ensure responsive interaction.        | Performance Requirement [6]        |
+| environmental and social responsibility      |  The system must work according to the environmental and social responsibility performance criteria for computers of the IEEE | IEEE Standard [5] |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/LCD Block Subsystem.md]: What does this mean precisely?

File: Documentation/Signoff/LCD Block Subsystem.md (hunk 1 above)
Comment (abdoulm366) [reply Documentation/Signoff/LCD Block Subsystem.md]: Depending on the power consumption, the brightness of the TFT LCD screens is generally between 200-400 nits.

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+| Description       | Constraints/Specification:                                                                           | Source                         |
+|-------------------|------------------------------------------------------------------------------------------------------|--------------------------------|
+| Visibility        | The LCD must provide clear visibility indoor and outdoor with a brightness level of at least 350 nits.                   |  Interaction Design principles [3]             |
+| User Interface    | The menu should allow easy navigation at any point with buttons such as "Return to Menu" or "Next" or "Back".    | User interface design [4]       |
+| Interactivity     | The system must register input from the keypad within 300ms to ensure responsive interaction.        | Performance Requirement [6]        |
+| environmental and social responsibility      |  The system must contain, on average, a minimum of 2% of any combination of postconsumer recycled plastic, ITE-derived post-consumer recycled plastic, or bio based plastic, measured as a percentage of total amount of plastic (by weight) in the product. | IEEE Standard [5] |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/LCD Block Subsystem.md]: This does not at all cover the requirements of that particular standard. If it is going to be used, it must be adhered to.

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+## Constraints/Specifications:
+
+| Description       | Constraints/Specification:                                                                           | Source                         |
+|-------------------|------------------------------------------------------------------------------------------------------|--------------------------------|
+| Visibility        | The LCD must provide clear visibility indoor and outdoor with a brightness level of at least 350 nits.                   |  Interaction Design principles [3]             |
+| User Interface    | The menu should allow easy navigation at any point with buttons such as "Return to Menu" or "Next" or "Back".    | User interface design [4]       |
+| Interactivity     | The system must register input from the keypad within 300ms to ensure responsive interaction.        | Performance Requirement [6]        |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/LCD Block Subsystem.md:13]: Based on what is this considered responsive?

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+## Subsystem Function:
+
+The purpose of this subsystem is to help ease the user's understanding of the kit's functionalities, provide a digital user manual, provide the users with instructions or guided learning via challenges and lessons and enable a digital interaction with the car. Hence, this LCD Block Subsystem serves as the one of the primary interaction point between the user and the device as a whole. 
+
+## Constraints/Specifications:
+
+| Description       | Constraints/Specification:                                                                           | Source                         |
+|-------------------|------------------------------------------------------------------------------------------------------|--------------------------------|
+| Visibility        | The LCD must provide clear visibility indoor and outdoor with a brightness level of at least 350 nits.                   |  Interaction Design principles [3]             |
+| User Interface    | The menu should allow easy navigation at any point with buttons such as "Return to Menu" or "Next" or "Back".    | User interface design [4]       |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/LCD Block Subsystem.md:12]: This doesn't sound easy to navigate

**Review by JesseTNRoberts — COMMENTED** [review]
You made a compelling argument for the inclusion of computing. Yet there is none included here?

**Comment by micahr234** [comment]
You need to remove Jesse and request a review from me.

File: Documentation/Signoff/LCD Block Subsystem.md
Comment (micahr234) [inline Documentation/Signoff/LCD Block Subsystem.md:1]: Sync with you team. Their PCBs need to be combined onto your board.

You have several spelling grammar mistakes. Reread and fix.

I am only reviewing the hardware for now. Software can be changed later.

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+| Interactivity     | The system must register input from the keypad within 300ms to ensure responsive interaction.        | Performance Requirement [6]        |
+| environmental and social responsibility      |  The system must contain, on average, a minimum of 2% of any combination of postconsumer recycled plastic, ITE-derived post-consumer recycled plastic, or bio based plastic, measured as a percentage of total amount of plastic (by weight) in the product. | IEEE Standard [5] |
+
+## Schematic:
+
+<p align="center">
+  <img height="500" src="https://github.com/user-attachments/assets/c6c65c65-3b30-4d90-81c9-5c3017dd02a9">
```
Comment (micahr234) [inline Documentation/Signoff/LCD Block Subsystem.md]: What does the power switch do? Is it wired correctly?

Should Vcc on the ultrasonic module be connected to a processor GPIO pin?

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+</p>
+
+## PCB
+
+<p align="center">
+  <img src="https://github.com/user-attachments/assets/4b75d724-e7eb-4ba1-96e5-97d4bab65dd8" height="300" /> <img src="https://github.com/user-attachments/assets/5c0e6445-8fc5-47dd-be11-3d47459fd45b" height="300" />
```
Comment (micahr234) [inline Documentation/Signoff/LCD Block Subsystem.md]: Why does your PCB show the Arduino Mega's footprint on it? You should just show connection pins.

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+
+The integrating of the LCD Block Subsystem is done by pairing a 3.2" TFT Touch Screen with a TFT LCD Arduino Mega Shield. A shield is needed since the TFT Touch screen operates at 3.3V while the Arduino Mega outputs at 5V. For the distance sensor, a HC-SR04 ultrasonic sensor is needed. While for the headlights, an RGB LED with three resistors are required. A push button is needed for the game and 2 Fans is needed for the A/C subsystem.
+
+Most of the testing shall be done within the code section and the hardware using LT-spice or other circuit assimilators. This includes unit tests for individual components, integration tests with the MCU and keypad, and system-level user acceptance tests to validate functionality, user interface design, and interactivity as per the defined requirements.
```
Comment (micahr234) [inline Documentation/Signoff/LCD Block Subsystem.md]: Testing is not done with LT Spice. Analysis is done with LT spice.

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+|-------------------------------------------------------------|---------------|---------------|------------|
+| 3.2″ TFT Touch Display                                      | 1             | Amazon.com    | $ 25.99    |
+| TFT Display Mega Shield                                     | 1             | Amazon.com    | $ 7.82     |
+| Arduino Mega Board                                          | 1             | Amazon.com    | $ 48.90    |
+| Ultrasonic Module HC-SR04                                   | 1             | Amazon.com    | $ 11.99    |
+| Multiple Colour Colorful Angel Eye Lamp RGB LED Lights 17MM | 2             | aliexpress.us | $ 14.74    |
+| RC Motor Cooling Fan                                        | 2             | Amazon.com    | $ 8.98     |
+| Resistors pack                                              | 1             | Amazon.com    | $ 2.80     |
+| Push On button pack                                         | 1             | digikey.com   | $ 1.97     |
+|                                                             |               | Total:        | $125.16    |
```
Comment (micahr234) [inline Documentation/Signoff/LCD Block Subsystem.md]: Need part number and manufacturer on BOM.

File: Documentation/Signoff/LCD Block Subsystem.md [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+ <h4 align="center"> Interact with car: </h4>
+This submenu “Interact with the car” will have 4 parts, the car’s proximity sensor which will display the closest object to the car, the car’s headlight control which will be able to change the car’s headlight color, the Birduino game which is an encoded game for the TFT LCD Touch Screen, the Fan ON/OFF which will turn on and off the car’s fan.
+
+<p align="center">
+  <img src="https://github.com/abdoulm366/TTU-Capstone--Electrical-Class-Kit/assets/125443044/a0fd1e55-b5d7-4bca-be09-41c6a98e0542" height="300" />  <img src="https://github.com/abdoulm366/TTU-Capstone--Electrical-Class-Kit/assets/125443044/9341daf4-6e7f-4299-8d2d-67c7e083972a" height="300" />  <img src="https://github.com/abdoulm366/TTU-Capstone--Electrical-Class-Kit/assets/125443044/4d54b888-ebdf-4eae-965e-a7d0b8475299" height="300" />
+</p>
+
+The first example is distance measuring with ultrasonic sensors. The sensor's output, or distance, is printed on the screen and will allow the user to pick the units, centimeters or inches, with the touch screen.
+
+The car’s headlight control shall be done by controlling an RGB LED with
```
Comment (micahr234) [inline Documentation/Signoff/LCD Block Subsystem.md]: I do not think the headlights are RGB.

**Review by micahr234 — APPROVED** [review]

**Review by micahr234 — APPROVED** [review]

**Pull request merged**